New Icons

baitmooth opened the pull request:
_ | _ | _ | _ | _
----- | ----- | ----- | ----- | ----- |
actual | antennapod | antimine | autotools | biglybt
birthdaydroid | boxcryptor | brackeyside | changelog | circuittrainer
conv6ations | cronograma | cuberite | drastic | filmstaden
financemanager | gardine | gpstest | helm | helmsync
hendroid | iss-detektor | kid3 | librespeed | look4sat moneymanagerex | mozcforandroid | myboy | myfiles | nclientv2
noice | oandbackupx | podcastaddict | portauthority | potatoprojectnotes
railplanner | ruler | secusoshoppinglist | shuttle | simplekeyboard
sknetrafiken | skymap | smartpack | solitairecg | spacelaunchnow
steamchat | sweb | terminalemulator | trafictransportsparisiens | units
ynab | 

![collection](https://user-images.githubusercontent.com/23211478/119182715-947a9900-ba73-11eb-9094-0b62c4560650.png)

baitmooth committed:
better svg

-removed groups
-added xmlns

baitmooth committed:
initial file

baitmooth committed:
added librespeed to appfilter.xml

baitmooth committed:
units.svg

baitmooth committed:
added xml

baitmooth committed:
applied square template

baitmooth committed:
created moneymanagerex.svg

baitmooth committed:
added look4sat.svg

baitmooth committed:
created shuttle+.svg

baitmooth committed:
rename

rename from other/shuttle+.svg
rename to other/shuttle.svg

baitmooth committed:
created conv6ations.svg

baitmooth committed:
created skymap.svg

baitmooth committed:
added kid3.svg

baitmooth committed:
added antimine.svg

baitmooth committed:
added antimine to appfilter.xml

baitmooth committed:
added conv6ations to appfilter.xml

baitmooth committed:
added Kid3 to appfilter.xml

baitmooth committed:
added Look4Sat to appfilter.xml

baitmooth committed:
added Money Manager Ex to appfilter.xml

baitmooth committed:
added Shuttle+ to appfilter.xml

baitmooth committed:
added Sky Map to appfilter.xml

baitmooth committed:
removed units.xml

baitmooth committed:
added Units to appfilter.xml

baitmooth committed:
created financemanager.svg

baitmooth committed:
added Finance Manager to appfilter.xml

baitmooth committed:
created shoppinglist.svg

baitmooth committed:
added Shopping List to appfilter.xml

baitmooth committed:
created solitairecg.svg

baitmooth committed:
added SolitaireCG to appfilter.xml

baitmooth committed:
Merge branch 'antimine' into icons

baitmooth committed:
Merge branch 'conv6ations' into icons

baitmooth committed:
Merge branch 'financemanager' into icons

baitmooth committed:
Merge branch 'kid3' into icons

baitmooth committed:
Merge branch 'librespeed' into icons

baitmooth committed:
Merge branch 'look4sat' into icons

baitmooth committed:
Merge branch 'moneymanagerex' into icons

baitmooth committed:
Merge branch 'solitairecg' into icons

baitmooth committed:
Merge branch 'shoppinglist' into icons

baitmooth committed:
Merge branch 'units' into icons

baitmooth committed:
Merge branch 'skymap' into icons

baitmooth committed:
Merge branch 'shuttle+' into icons

baitmooth committed:
added librespeed to appfilter.xml

baitmooth committed:
deleted other/appfilter.xml

baitmooth committed:
codecleanup of skymap.svg

baitmooth committed:
some more optimizations in skymap.svg

baitmooth committed:
fixed error in <style>tag

baitmooth committed:
added f-droids 'corona contact tracing germany' to use the existing
cwa-icon

baitmooth committed:
created gpstest.svg

baitmooth committed:
added GPSTest

baitmooth committed:
Merge branch 'gpstest' into 'icons'

baitmooth committed:
new svg's

dkanada/frost/issues/119
DraStic		drastic.svg
Filmstaden	filmstaden.svg
My Boy		myboy.svg
My Files	myfiles.svg
Skånetrafiken	sknetrafiken.svg

baitmooth committed:
updated svgs

removed inkscape tags from myboy.svg, myfiles.svg, sknetrafiken.svg

rounded square for myfiles.svg

baitmooth committed:
6 new svg's

YNAB
Space Launch Now
Simple Keyboard
Rail planner
Port Authority
Podcast Addict

baitmooth committed:
updated svg's

part of dkanada/issue/245

baitmooth committed:
created autotools.svg

baitmooth committed:
stripped with scour

baitmooth committed:
added new icons to appfilter.xml

AutoTools, DraStic, Filmstaden, My Boy!, My Files, Podcast Addict, Port
Authority, Rail planner, Simple Keyboard, Skånetrafiken, Space Launch
Now, YNAB

baitmooth committed:
more icons for #245

baitmooth committed:
added boxcryptor, ISS-Detektor

baitmooth committed:
stripped boxcryptor

baitmooth committed:
created ruler, steamchat

baitmooth committed:
added ruler, steamchat

baitmooth committed:
5 new icons for #328

baitmooth committed:
updated appfilter.xml

baitmooth committed:
created cronograma, circuittrainer #336

baitmooth committed:
added circuittrainer cronograma

baitmooth committed:
4 new icons

some from #351

baitmooth committed:
added BiglyBT

baitmooth committed:
code changed for Banque #379

baitmooth committed:
3 new icons

dkanada commented:
Some general thoughts about the current icons are below. Mostly about making them simpler and more uniform, the general designs look fine to me.

- playing card icon should be a single rectangular card for simplicity
- the big `F` icon doesn't match the other circle sizes
- shuttle icon looks too complicated with that folded middle section (can simplify without removing the fold)
- simple steam icon should already exist and should be used for the steam chat icon
- MyBoy and the ISS icon are too complicated but I have no clue how to make them simpler
- punch out the circle in the spaceship icon
- unwrap the inner objects in the files and Cronograma icons (the cronograma icon can remove the border circle for example)

baitmooth committed:
remake of some icons

cronograma: removed outer circle
filmstaden: fixed size
myboy: drastic simplified
shuttle: simplified fold
solitiarecg: reduced number of cards to one, resize
spacelaunchnow: changed window
steamchat: used simple steamlogo

baitmooth commented:
Thanks for ideas and hints.

- [x] playing card icon should be a single rectangular card for simplicity
- [x] the big `F` icon doesn't match the other circle sizes
- [x] shuttle icon looks too complicated with that folded middle section (can simplify without removing the fold)
- [x] simple steam icon should already exist and should be used for the steam chat icon
- [x]  MyBoy icon are too complicated 

- tried a drastic version

- [ ]  ISS icon are too complicated 
- [x] punch out the circle in the spaceship icon
- [ ]  unwrap the inner objects in the files and Cronograma icons (the cronograma icon can remove the border circle for example)
- uncircled the cronograma icon, and will see what can be unwrapped next.

dkanada commented:
Another quick mention, I would use an existing settings icon for the AutoTools icon, and figure out a way to increase the outline for the wrench tool. It's too thin right now and won't show up on low DPI screens.

baitmooth commented:
> Another quick mention, I would use an existing settings icon for the AutoTools icon, and figure out a way to increase the outline for the wrench tool. It's too thin right now and won't show up on low DPI screens.

Ok, thanks for the hint. I will try this. I will also redo Rail planner and Helm Sync which are similar with thin lines.

baitmooth committed:
updated 4 icons

- autotools to make the wrench more visible
- helmsync for simplicity
- railplanner made the train more solid
- solitairecg changed hearts for spades

baitmooth committed:
autotools again

baitmooth committed:
new icons, updated appfilter

new (svg & appfilter):
BirthDayDroid
Changelog
Notes
OAndBackupX
SmartPack

updated (appfilter):
Gestion des Ports
Open Food Facts

baitmooth committed:
changed icon name

changed shoppinglist to shopping-list to prevent conflict

baitmooth commented:
There are now two apps named 'Shopping List' they seem to be different apps which may be installed alongside. I renamed the one from secuso. And the question is a little bit about naming... are dashes allowed, like 'shopping-list'? Would an name like 'secusoshoppinglist' be better?

baitmooth committed:
deleted shoppinglist.svg

dkanada commented:
I don't believe dashes are allowed so `secusoshoppinglist` would be best.

baitmooth committed:
removed dashes from icon names

shopping-list -> secusoshoppinglist
iss-detektor -> issdetektor

baitmooth commented:
> I don't believe dashes are allowed so `secusoshoppinglist` would be best.

Done.

dkanada commented on app/src/main/res/xml/appfilter.xml:
```diff
@@ -311,7 +320,7 @@
     <item component="ComponentInfo{nl.asnbank.asnbankieren/com.service2media.m2active.client.android.M2ActiveClient}" drawable="bankieren" />
 
     <!-- Banque -->
-    <item component="ComponentInfo{com.caisseepargne.android.mobilebanking/com.caisseepargne.android.mobilebanking.activities.ADashBoard}" drawable="banque" />
+    <item component="ComponentInfo{com.caisseepargne.android.mobilebanking/com.caisseepargne.android.mobilebanking.features.main.MainActivity}" drawable="banque" />
```
Generally I keep both so people with old apps still get the icons.

baitmooth replied:
I get the idea. Fixed it.

baitmooth committed:
brackeyside, birthdaydroid, biglybt appfilter

brackeyside -> new icon, update in appfilter
biglybt -> file updated
birthdaydroid -> file updated

baitmooth committed:
reworked issdetektor

baitmooth committed:
reworked myfiles.svg

baitmooth committed:
added gardine.svg updated appfilter.xml #390

dkanada commented:
Wow you are really going through a lot of the requests :smile: I have a few comments, but at this point all of the icons can be merged, I'm just commenting on things that could be improved.

- HelmSync looks a bit larger than the other circles
- CircuitTrainer could maybe utilize one of the existing clock or person icons to better match the rest of the pack
- BoxCryptor might look good with less parts and larger gaps between them (maybe a lock icon somewhere)
- BiglyBT could somehow be simplified (the inner left curve also doesn't look entirely parallel)

dkanada commented:
If you'd rather merge this as is though, I will be sure this gets in the next release.

baitmooth commented:
> Wow you are really going through a lot of the requests smile I have a few comments, but at this point all of the icons can be merged, I'm just commenting on things that could be improved.

It's a fun way to spend time ;) while finally gaining inkscape-skills.

>     * HelmSync looks a bit larger than the other circles
>
I'm not completly happy with Helm/HelmSync but i have not figured out how to really improve it. It only looks bigger, and it is actually a few picopixel smaller. 
 
>     * CircuitTrainer could maybe utilize one of the existing clock or person icons to better match  the rest of the pack
> 
>     * BoxCryptor might look good with less parts and larger gaps between them (maybe a lock icon somewhere)
> 
>     * BiglyBT could somehow be simplified (the inner left curve also doesn't look entirely parallel)

Good ideas, thanks for that. I will put it in my tasklist for rework.

>  If you'd rather merge this as is though, I will be sure this gets in the next release.

I have no clear preference about the time of merge. Would be happy to see some soon, but no hurry about that. I take this as a moment for kind of feature freeze and looking at the issues.

baitmooth committed:
rework of 3 icons

biglybt - was not very symmetric
boxcryptor - too small segments
circuittrainer - changed that running person

baitmooth committed:
updated circuittrainer

changed the clock

dkanada commented on app/src/main/res/xml/appfilter.xml:
```diff
@@ -518,6 +540,12 @@
     <!-- Change Detection -->
     <item component="ComponentInfo{com.bernaferrari.changedetection/com.bernaferrari.changedetection.MainActivity}" drawable="changedetection" />
 
+    <!-- Changelog -->
+    <item component="ComponentInfo{com.bytehamster.changelog/com.bytehamster.changelog.Main}" drawable="changelog" />
+
+    <!-- Changelog -->
+    <item component="ComponentInfo{com.bytehamster.changelog/com.bytehamster.changelog.Main}" drawable="changelog" />
```
probably only want one of these :)
